CRUD_operations.c

```diff
@@ -0,0 +1,131 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include<stdbool.h>
+typedef struct USER {

```

Comment:
for structure, typedef use namingconvention: 

Use PascalCase (capitalize the first letter of each word) for struct and typedef names.
Use descriptive names that reflect the data structure's purpose.


typedef struct Student {
    int id;
    char name[50];
    float grade;
} Student;

Use PascalCase (capitalize the first letter of each word) for struct and typedef names.
Use descriptive names that reflect the data structure's purpose.

ex:
typedef struct Student {
    int id;
    char name[50];
    float grade;
} Student;

Student student_data;